File: examples/FaxSend.cs
using System;
using System.Collections.Generic;
using System.IO;
using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

public class Example
{
    public static void Main()
    {
        var config = new Configuration();
        config.Username = "YOUR_API_KEY";

        var faxApi = new FaxApi(config);

        var files = new List<Stream> {
            new FileStream(
                "./example_fax.pdf",
                FileMode.Open,
                FileAccess.Read,
                FileShare.Read
            )
        };

        var data = new FaxSendRequest(
            files: files,
            testMode: true,
            recipient: "16690000001",
            sender: "16690000000",
            coverPageTo: "Jill Fax",
            coverPageMessage: "I'm sending you a fax!",
            coverPageFrom: "Faxer Faxerson",
            title: "This is what the fax is about!",
        );

        try
        {
            var result = faxApi.FaxSend(data);
            Console.WriteLine(result);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Dropbox Sign API: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: examples/FaxSend.py
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, apis, models

configuration = Configuration(
    # Configure HTTP basic authorization: api_key
    username="YOUR_API_KEY",
)

with ApiClient(configuration) as api_client:
    fax_api = apis.FaxApi(api_client)

    data = models.FaxSendRequest(
        files=[open("example_signature_request.pdf", "rb")],
        test_mode=True,
        recipient="16690000001",
        sender="16690000000",
        cover_page_to="Jill Fax",
        cover_page_message="I'm sending you a fax!",
        cover_page_from="Faxer Faxerson",
        title="This is what the fax is about!",
    )

    try:
        response = fax_api.fax_send(data)
        pprint(response)
    except ApiException as e:
        print("Exception when calling Dropbox Sign API: %s\n" % e)
